controllers/buildpipeline/buildpipeline_controller_test.go

```diff
@@ -41,14 +41,14 @@ import (
 
 var _ = Describe("PipelineController", func() {
 	var (
-		manager            ctrl.Manager
-		pipelineReconciler *Reconciler
-		scheme             runtime.Scheme
-		req                ctrl.Request
-		successfulTaskRun  *tektonv1beta1.TaskRun
-		testpipelineRun    *tektonv1beta1.PipelineRun
```

Comment:
Why is this change needed?

Reply:
I mean renaming `test*` to `integration*`

Reply:
One is build pipelinerun, another is integration test pipelinerun, if we still use the old name, it looks weird and confusing, so I would change their name.

Reply:
okay